src/blas/highlevel.jl

```diff
@@ -47,8 +47,19 @@ end
 LinearAlgebra.BLAS.asum(x::ROCBLASArray) = asum(length(x), x, 1)
 
 function LinearAlgebra.axpy!(alpha::Number, x::ROCArray{T}, y::ROCArray{T}) where T<:ROCBLASFloat
```

Comment:
```suggestion
function LinearAlgebra.axpy!(alpha::Number, x::ROCArray{T}, y::ROCArray{T}) where T<:Union{ROCBLASFloat,ROCBLASComplex}
```
Has a fallback in GPUArrays, but let's not hit that.